### DISCUSSION TITLE
QC metrics

### DISCUSSION THREAD

**Discussion by l-espana** [ExploreASL/ExploreASL#1500] [Q&A] [answered]
Hello!
Just wondering if there's a list anywhere about the various reported QC metrics that get saved to PDF? For example, Mean_SSIM_Perc and RMSE_perc are a little unclear and I'm not sure where to find the information on what these metrics are referring to.
Thanks!

**Comment by maartenhammer** [comment, marked as answer]
Hello Lezlie, 
As part of [the ExploreASL NeuroImage Paper](https://www.sciencedirect.com/science/article/pii/S1053811920305176) this information is available in the Appendix A. [Supplementary Data](https://ars.els-cdn.com/content/image/1-s2.0-S1053811920305176-mmc1.pdf).
I hope this helps you!

For these specific metrics it states the following:
Mean_SSIM_Perc % Mean voxel-wise structural similarity index (Wang et al., 2004) (higher = better)
nRMSE_Perc % nRMSE with ASL template (lower = better)
